src/Microsoft.AspNetCore.SignalR.Core/HubConnectionContext.cs

```diff
@@ -312,6 +331,8 @@ public virtual void Abort()
                 return;
             }
 
+            _connectedAborted = false;
```

Comment:
This is backwards now right? If the name is `_connectionAborted`, it should be being set to `true` here :)